### ISSUE TITLE
Unread message in Channel Review wrong when using channel. sendMessage

### ISSUE THREAD

**Issue by vihan85** [GetStream/stream-chat-react#2217]
**Describe the bug**

I created a custom action Forward the message and use "channel.sendMessage" to send a message.

```
 channel.sendMessage({
text: messageForward.text,
attachments: messageForward.attachments,
});
```

1. Channel A has two messages unread.
2. From channel B use channel.sendMessage by custom action Forward Message to send a message to channel A
3. Reload

I would like to keep an unread number after using channel.message

**Screenshots**

https://github.com/GetStream/stream-chat-react/assets/104440194/425e16ac-8984-4223-ad8a-b92024842eb3

**Package version**
- stream-chat-react: 11.1.1
- stream-chat-css: 4.1.0
- stream-chat-js: 8.14.4

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by MartinCupela** [comment]
@vihan85 do I understand it well, that the bug consists in unread count not being increased when a new message arrives?

Could you please confirm the following?

1. Does the user Vi Han receive `message.new` event via the WS for the message "3"?
2. Are there any custom components / logic involved in your implementation of `ChannelList`?

Thank you

**Comment by minhth-1529** [comment]
Hi @MartinCupela The problem is not unread count do not increase but unread count disappear after using the channel.sendMess function, use the channel.sendMess function to send the message, not activate that channel.

**Comment by MartinCupela** [comment]
Thank you for the clarification @minhth-1529 . 

Would you be so kind and create a minimal reproduction by forking [this Codesandbox](https://codesandbox.io/p/devbox/support-to-fork-template-g64slj)? That way we will be sure we are debugging relevant code. Thank you

**Comment by minhth-1529** [comment]
Hi @MartinCupela, I am already done Forward message in you Codesandbox, but I can not demo, because I need this user who I am talking reply and I don't read that message, aim to unread number appear. Now I chat one way, that random account can not reply

**Comment by minhth-1529** [comment]
![image](https://github.com/GetStream/stream-chat-react/assets/17897146/a9dac508-f963-46cc-b3ff-1b6970165a8e)
Ex: I need to General reply, so unread count appear, after that I will Forward message with Talk about React to General, then unread count on General will disappear

**Comment by MartinCupela** [comment]
@minhth-1529 you can use URL search params to customize, which user is connected. An example from a Vite app:

```
const params = (new Proxy(new URLSearchParams(window.location.search), {
  get: (searchParams, property) => searchParams.get(property as string),
}) as unknown) as Record<string, string | null>;

const apiKey = import.meta.env.VITE_STREAM_KEY as string;
const userId = params.uid ?? (import.meta.env.VITE_USER_ID as string);
const userToken = params.ut ?? (import.meta.env.VITE_USER_TOKEN as string);

const filters: ChannelFilters = { members: { $in: [userId] }, type: 'messaging' };
const options: ChannelOptions = { limit: 10, presence: true, state: true };
const sort: ChannelSort = { last_message_at: -1, updated_at: -1 };
``

**Comment by vihan85** [comment]
hi @MartinCupela 
I am already custom action Forward, you can find out and do it step by step, as @minhth-1529  said.

[Codesandbox](https://codesandbox.io/p/devbox/support-to-fork-template-forked-pzn82f?file=%2Fsrc%2FApp.tsx%3A41%2C31)

**Comment by MartinCupela** [comment]
Thank you @vihan85 , I am looking into this.

**Comment by MartinCupela** [comment]
This issue has been fixed with [v11.5.0](https://github.com/GetStream/stream-chat-react/releases/tag/v11.5.0)
